asst1/asst1.py

```diff
@@ -0,0 +1,74 @@
+# Assignment 1: How to Python
+
+from collections import Counter
+import sys
+
+
+### swapchars
+def swap(string, i, j):
```

Comment:
You only ever use this function "swap" within "swapchars" and therefore it should be scoped as such. In Python, you can actually declare functions *within* other functions, which is pretty dope